advisor/src/main/kotlin/advisors/OssIndex.kt

```diff
@@ -132,7 +132,7 @@ class OssIndex(name: String, serverUrl: String = OssIndexService.DEFAULT_BASE_UR
         coordinates: List<String>
     ): List<OssIndexService.ComponentReport> =
         try {
-            logger.debug { "Querying component report from ${OssIndexService.DEFAULT_BASE_URL}." }
+            logger.debug { "Querying component report." }
```

Comment:
Out of scope: 
- maybe we should rename coordinates to e.g. purls for clarify.
- maybe move the output to the caller, which could then at least tell "Querying component reports chunk $i of $n."

Reply:
> maybe we should rename coordinates to e.g. purls for clarity.

Good idea, done in a new commit.

> maybe move the output to the caller, which could then at least tell "Querying component reports chunk $i of $n."

I like that also, done.